liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -888,7 +888,10 @@ private Xpp3Dom getPluginConfig(Plugin plugin, Xpp3Dom overrides) {
     
     private static final ArrayList<String> commonServerParams = new ArrayList<>(Arrays.asList(
             "serverXmlFile", "configDirectory", "bootstrapProperties", "bootstrapPropertiesFile",
-            "jvmOptions", "jvmOptionsFile", "serverEnvFile"));
+            "jvmOptions", "jvmOptionsFile", "serverEnvFile",
+            // alias parameters
+            "configFile"
```

Comment:
serverEnv is also an alias for serverEnvFile

Reply:
And runtimeArtifact has an alias assemblyArtifact, and runtimeArchive has an alias assemblyArchive.

Reply:
thanks!  I will add them